biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -1326,8 +1334,13 @@ private void doNamesection(Jar dot, Manifest manifest) {
 				String path = i.next();
 				// For each resource
 
-				if (instr.getKey()
-					.matches(path)) {
+				Instruction instruction = instr.getKey();
+
+				if (path.endsWith("/") && (!instruction.isLiteral() || !instruction
```

Comment:
What about allowing wildcards for packages if the wildcard ends in `/`? Then you could do `-namesection: javax.*/;sealed=true`. I can see that you do not want wildcards not ending in slash to apply to package entries. But I also think we should support wildcards ending in slash to apply to matching package entries?

Reply:
Yeah this occurred to me as well. I'll make the change.